doc/markdownlinkconverter/markdownlinkconverter.c

```diff
@@ -0,0 +1,452 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <stdbool.h>
+#include <string.h>
+
+#if defined(WIN32)
+#define FOLDER_DELIMITER '\\'
+#else
+#define FOLDER_DELIMITER '/'
+#endif
+
+/* Slashes or backslashes to the elektra root.
+ * Depends on the place of the executable, example:
+ * /foo/bar/elektra/build/bin/executable
+ * E:\\foo\bar\elektra\build\bin\executable
+ * In both cases the there are 3 slashes or backslashes
```

Comment:
I think there is no possibility (without a hack) to pass an extra argument to the input filter, it will be called by popen with <filter> <input-file> (https://github.com/doxygen/doxygen/blob/389dc4c06c31a5da26a55bf2f6a8ba8316dbc1ff/src/definition.cpp#L747). But there is the possibility to write the project directory path in a file with cmake, read it with the input filter and delete it after the doxygen call.